rwengine/src/script/modules/GameModule.cpp

```diff
@@ -66,6 +66,18 @@ void game_print_now(const ScriptArguments& args)
 					));
 }
 
+void game_print_soon(const ScriptArguments& args)
+{
+	std::string id(args[0].string);
+	std::string str = args.getWorld()->data->texts.text(id);
+	int time = args[1].integer;
+	unsigned short style = args[2].integerValue(); // same as flags?!
+	args.getWorld()->state->text.addText<ScreenTextType::BigLowPriority>(
+				ScreenTextEntry::makeBig(
+					id, str, style, time
+					));
+}
+
```

Comment:
Weird indentation.

We don't have a style standard for this type of thing, I think. Maybe something like:
```
args.getWorld()->state->text.addText<ScreenTextType::BigLowPriority>(
	ScreenTextEntry::makeBig(
		id, str, style, time
	)
);
```
or because the parameters are short
```
args.getWorld()->state->text.addText<ScreenTextType::BigLowPriority>(
	ScreenTextEntry::makeBig(id, str, style, time)
);
```